### PR TITLE
fix local layer padding docstring

### PR DESCRIPTION
**Pull request by joelthchao**
#5925 Fix docstring. Both `LocallyConnected1D` and `LocallyConnected2D` support only `valid` for padding.

### CODE REVIEW

File: keras/layers/local.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ class LocallyConnected1D(Layer):
             specifying the stride length of the convolution.
             Specifying any stride value != 1 is incompatible with specifying
             any `dilation_rate` value != 1.
-        padding: One of `"valid"` or `"same"` (case-insensitive).
+        padding: `"valid"` (case-insensitive).
```
Comment (fchollet) [inline keras/layers/local.py]: If there is only one possible value then there shouldn't be an argument. Removing the argument would cause further backwards compatibility issues, so instead please just add a sentence explaining that more values could be added in the future but that only one value is supported for now.

**Comment by MarkSeygan** [comment]
May I ask what is problematic with implementing ` 'same'` border_mode for Locally connected convs? Or, is there some non-technical issue with it? I kinda think that if I pad my training data with zeroes according to kernelsize, then that would be it because I use it as the first layer.

**Comment by joelthchao** [comment]
@MarkSeygan The actual way to implementing `same` padding is calculating required zero padding and do the locally connected layer operation. Contribution is welcome!
